clang/compiler-course/ionova_ekaterina_lab_1/CMakeLists.txt

```
@@ -0,0 +1,14 @@
+add_llvm_library(ClassInfoVisitorPlugin MODULE ClassInfoPlugin.cpp PLUGIN_TOOL clang)
```

Feel free to sign the plugin with your data:
```suggestion
set(Title "ExamplePlugin")
set(Student "Ionova_...")
set(Group "...")
set(TARGET_NAME "${Title}_${Student}_${Group}_ClangAST")
add_llvm_library(ClassInfoVisitorPlugin MODULE ClassInfoPlugin.cpp PLUGIN_TOOL clang)
```